srcpkgs/alsa-ucm-conf/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'alsa-ucm-conf'
+pkgname=alsa-ucm-conf
+version=1.2.4
+revision=1
+archs="i686* x86_64*"
```

Comment:
hm, why is it restricted?

Reply:
My thought was originally that it would only be useful for x86 chips, but I looked into it more and it provides files that are useful on a variety of architectures. I'll remove the line.